psi4/driver/json_wrapper.py

```diff
@@ -50,9 +49,132 @@
 
 
 def run_json(json_data):
+
+    # Set scratch
+    if "scratch_location" in json_data:
+        psi4_io = core.IOManager.shared_object()
+        psi4_io.set_default_path(json_data["scratch_location"])
+
+    # Set memory
+    if "memory" in json_data:
+        psi4.set_memory(json_data["memory"])
+
+    # Do we return the output?
+    return_output = json_data.pop("return_output", False)
+    if return_output:
+        outfile = os.path.join(core.IOManager.shared_object().get_default_path(), str(uuid.uuid4()) + ".json_out")
+        core.set_output_file(outfile, False)
+        json_data["raw_output"] = "Not yet run."
+
+    # Set a few flags
+    json_data["raw_output"] = None
+    json_data["success"] = False
+
+    # Attempt to run the computer
+    try:
+        if ("schema_name" in json_data) and (json_data["schema_name"] == "QC_JSON"):
+            run_json_qc_schema(json_data)
+        else:
+            run_json_original_v1_1(json_data)
+
+    except Exception as error:
+        json_data["error"] = repr(error)
+        json_data["success"] = False
+
+    if return_output:
+        with open(outfile, 'r') as f:
+            json_data["raw_output"] = f.read()
+        os.unlink(outfile)
+
+    return json_data
+
+
+def run_json_qc_schema(json_data):
```

Comment:
Am I overlooking keywords/options setting in qc_schema as opposed to original_v1_1?

Reply:
The `keywords` field is currently a free for all according to the spec?

Reply:
But if present, we need to set them, yes? I'll add something, then see if we're at cross-purposes.